### PR TITLE
Set up GitHub pages for website

### LINKED ISSUE #12: Setup website for this repo
More translations are going to be provided, we will need to set up the website, maybe GitHub page with some theme.

### PR DESCRIPTION
**Pull request by kezhenxu94**
The branch name `gh-pages` is used as a deploy branch, so I open another PR. resolve #12

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  target_branch: gh-pages
+  name: kezhenxu94
+  email: kezhenxu94@apache.org
+  on:
+    branch: master
+
+notifications:
+  email:
+    recipients:
+      - kezhenxu94@apache.org
```
Comment (kezhenxu94) [inline .travis.yml]: FYI @wu-sheng @KAMIJYOUDOUMA @thelight1 , if you'd like to recieve notifications, feel free to add your email address here

**Comment by wu-sheng** [comment]
Let's try once:) @KAMIJYOUDOUMA @thelight1 Send pull request if you want to add your names.

**Comment by kezhenxu94** [comment]
works well, and I just put the URL to the home page of the repo
image

**Comment by wu-sheng** [comment]
Yes. Noticed :) Thanks so much.

**Pull request merged**